src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx

```diff
@@ -93,14 +92,14 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
     );
 
     this.cy = cytoscape(opts);
-    this.panzoom = this.cy.panzoom();
+    // Receives object with default init values or 'destroy' command and returns cy instance
+    this.cy.panzoom();
+    this.cy.container().querySelector('.cy-panzoom').style.zIndex = -1;
```

Comment:
do we want to change z-index?

Reply:
Yes, if we don’t, the panzoom will be over the left menu when scrolling and that looks wrong

Reply:
But the graph also goes on top of the controls :/
I'm ok with that if you see it right.

Reply:
Didn’t notice that. Will see if can workaround that.